app/page.tsx

```diff
@@ -1,54 +1,58 @@
-import SignUpUserSteps from "@/components/tutorial/SignUpUserSteps";
-import Header from "@/components/Header";
+"use client"
+import Link from "next/link";
+import { Caveat } from 'next/font/google';
+import { getUserEmail } from "@/utils/scripts/accountFunctions";
+import { useEffect, useState } from "react";
 
-export default async function Index() {
-  const canInitSupabaseClient = () => {
-    // This function is just for the interactive tutorial.
-    // Feel free to remove it once you have Supabase connected.
-    try {
-      createClient();
-      return true;
-    } catch (e) {
-      return false;
-    }
-  };
+const caveat = Caveat({ subsets: ['latin'] })
 
-  const isSupabaseConnected = canInitSupabaseClient();
+export default function Index() {
+  const [email, setEmail] = useState("");
+
+  useEffect(() => {
+    getUserEmail().then((res) => {
+      setEmail(res || "");
+    });
+  }, [])
 
   return (
-    <div className="flex-1 w-full flex flex-col gap-20 items-center">
-      <nav className="w-full flex justify-center border-b border-b-foreground/10 h-16">
-        <div className="w-full max-w-4xl flex justify-between items-center p-3 text-sm">
-          <DeployButton />
-          {isSupabaseConnected && <AuthButton />}
+    <div className="2xl:bg-[url('/assets/backgrounds/homeBG_desktop.svg')] bg-[url('/assets/backgrounds/homeBG_laptop.svg')] bg-cover bg-center bg-no-repeat min-h-screen min-w-full relative flex justify-center items-center">
+      {/* Top Nav Bar */}
+      <div className="absolute top-0 w-screen 2xl:h-28 h-20 flex flex-row text-wrap">
+        {/* Minted Logo */}
+        <div className="2xl:pt-6 pt-4 2xl:pl-10 pl-6"><Link href="/" className={`${caveat.className} 2xl:text-6xl text-5xl text-black font-medium`}>Minted</Link></div>
+        {/* Header Tabs */}
+        <div className="2xl:px-40 px-20 2xl:space-x-20 space-x-12 flex flex-row text-2xl text-heading items-center">
+          <Link href="/about">About</Link>
+          <Link href="/how_it_works">How It Works</Link>
+          <Link href="/why_minted">Why Minted</Link>
+        </div>
+        {/* Log In and Dashboard Button */}
+        <div className="flex flex-row flex-1 text-heading justify-end items-center">
+          {email && <div className="2xl:pr-10 pr-6 text-xl font-medium">Hey, {email}!</div>}
+          {email ?
+          <Link href="/dashboard" className="2xl:mr-10 mr-6 2xl:h-16 h-12 2xl:w-40 w-32 border-4 border-darkmaroon border-solid rounded-full text-darkmaroon 2xl:text-xl text-lg font-semibold flex justify-center items-center hover:bg-darkmaroon hover:text-primary transition duration-300">Dashboard</Link>
+          :
+          <Link href="/login" className="2xl:mr-10 mr-6 2xl:h-16 h-12 2xl:w-40 w-32 border-4 border-darkmaroon border-solid rounded-full text-darkmaroon 2xl:text-xl text-lg font-semibold flex justify-center items-center hover:bg-darkmaroon hover:text-primary transition duration-300">Log In</Link>
```

Comment:
Border width and font weight is too heavy for smaller screens. Border-2 and font-medium/normal prob better for smaller screens.

Button is too large for larger screen imo looks fat. Decrease height would make it look more proportional.